File: LV1/1.6/leftbeehind/leftbeehind.hs
solve :: (Num a, Ord a) => a -> a -> String
solve a b | a + b == 13 = "Never speak again."
          | a == b      = "Undecided."
          | a > b       = "To the convention."
          | otherwise   = "Left beehind."

forever :: IO ()
forever = do
  [a, b] <- map read . words <$> getLine
  if a == 0 && b == 0
    then return ()
    else do
      putStrLn $ solve a b
      forever

main :: IO ()
main = forever

File: LV1/1.6/leftbeehind/leftbeehind.go
package main

import "fmt"

func main() {
	var a, b int

	for {
		fmt.Scanf("%d %d", &a, &b)
		if a == 0 && b == 0 {
			break
		}

		switch {
		case a+b == 13:
			fmt.Println("Never speak again.")
		case a == b:
			fmt.Println("Undecided.")
		case a > b:
			fmt.Println("To the convention.")
		default:
			fmt.Println("Left beehind.")
		}
	}
}
